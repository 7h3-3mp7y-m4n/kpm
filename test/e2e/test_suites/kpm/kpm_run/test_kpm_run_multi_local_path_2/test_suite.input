kpm run <workspace>/pkg1