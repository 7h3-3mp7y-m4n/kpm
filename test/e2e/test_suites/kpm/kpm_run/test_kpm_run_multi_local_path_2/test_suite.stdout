adding 'pkg2'
The_first_kcl_program: Hello World!